### ISSUE TITLE
Windows client cannot find KBFS client after reboot

### ISSUE THREAD

**Issue by jonpavelich** [keybase/client#4921]
After rebooting my laptop (Windows 10), the K:/ drive in Windows Explorer would not load. I quit and reopened Keybase, and received an error message stating the KBFS client wasn't found.

Error message:
`Error: KBFS client wasn't found at t.Error (native) at new t (file:///C:/Users/<myusername>/AppData/Roaming/Keybase/Gui/resources/app/desktop/dist/index.bundle.js:11:5254) at a (file:///C:/Users/<myusername>/AppData/Roaming/Keybase/Gui/resources/app/desktop/dist/index.bundle.js:11:4885) at o (file:///C:/Users/<myusername>/AppData/Roaming/Keybase/Gui/resources/app/desktop/dist/index.bundle.js:11:4810) at file:///C:/Users/<myusername>/AppData/Roaming/Keybase/Gui/resources/app/desktop/dist/index.bundle.js:5:23565 at file:///C:/Users/<myusername>/AppData/Roaming/Keybase/Gui/resources/app/desktop/dist/index.bundle.js:53:23817 at e.t.Deferrals.e._call (file:///C:/Users/<myusername>/AppData/Roaming/Keybase/Gui/resources/app/desktop/dist/index.bundle.js:35:1893) at file:///C:/Users/<myusername>/AppData/Roaming/Keybase/Gui/resources/app/desktop/dist/index.bundle.js:35:2044 at c (file:///C:/Users/<myusername>/AppData/Roaming/Keybase/Gui/resources/app/desktop/dist/index.bundle.js:35:1671) at e.t.Deferrals.e._fulfill (file:///C:/Users/<myusername>/AppData/Roaming/Keybase/Gui/resources/app/desktop/dist/index.bundle.js:35:2003)` (my windows username was substituted for `<myusername>` in the above error).

my log id: 181f57dbc72ac01eacb8551c

**Comment by zanderz** [comment]
I see you ran an installer on 11/20, and that your logs seem to have run through 11/21 at 12:05. However, kbfs logs go to 11/21 at about 1:05. Not sure when you rebooted in there, but kbfsdokan.exe seems to be missing when you sent those logs. Can you please try the following:
- Reboot and send logs again
- If k: still doesn't appear, open keybase shell and enter `kbfsdokan -debug -mount-from-service` and post the results

**Comment by jonpavelich** [comment]
It is possible I did not reboot in that time. I thought I had last night, but I may have just put my laptop to sleep instead. Sorry for the confusion.

I rebooted, and the issue persisted with the same error message. Unfortunately before seeing your response requesting I run the `kbfsdokan` command, I re-ran the Windows installer and chose to Repair. This resolved the issue. If logs from this would be useful, I can submit them. Thank you.
